doc/BuildingAndRunning.md

```diff
@@ -54,6 +62,11 @@ The above instructions create an unoptimized debug build. The `--distribute` fla
 
 ## Building on Windows
 
+You should be able to build hermes by running the following script from the base of the repository.
+Note: There may be errors shown on the script but hermes is built if you can find the executable in build/bin/hermes/Debug/hermes.exe
+    
+    .ado\scripts\cibuild.ps1
+
 The Windows build depends on which particular combination of GitBash/Cygwin/WSL and Visual Studio is used.
 
     git -c core.autocrlf=false clone https://github.com/facebook/hermes.git
```

Comment:
not sure what these instructions are for, is this still an accurate way to build Hermes on windows or should it be deleted?